proto/viam/app/data/v1/data.proto

```diff
@@ -80,6 +79,24 @@ message Filter {
   repeated string org_ids = 11;
   repeated string mime_type = 12;
   CaptureInterval interval = 13;
+  TagsFilter tags_filter = 14;
+  repeated string tags = 5 [deprecated = true];
+}
+
+message TagsFilter {
+  TagsFilterType type = 1;
+  repeated string tags = 2; // tags are used to match documents (using logical OR) if the TagsFilterType is TAGS_FILTER_TYPE_MATCH_BY_OR
```

Comment:
nit: Since long comment, can move into its own line above, and might edit to be a bit shorter:

``` Tags are used to match documents if `type` is UNSPECIFIED or MATCH_BY_ORG```

Reply:
done